pandas/io/tests/parser/common.py

```diff
@@ -1469,3 +1469,23 @@ def test_memory_map(self):
 
         out = self.read_csv(mmap_file, memory_map=True)
         tm.assert_frame_equal(out, expected)
+
+    def test_read_csv_utf_aliases(self):
+        # see gh issue 13549
+        path = 'test.csv'
+        expected = DataFrame({'A': [0, 1], 'B': [2, 3],
+                              'multibyte_test': ['testing123', 'bananabis'],
+                              'mb_nums': [154.868, 457.8798]})
+
+        for byte in [8, 16]:
+            expected.to_csv(path, encoding='utf-' + str(byte), index=False)
+            for fmt in ['utf-{0}', 'utf_{0}', 'UTF-{0}', 'UTF_{0}']:
+                encoding = fmt.format(byte)
+                for engine in ['c', 'python', None]:
+                    out = self.read_csv(
```

Comment:
oh i c. we almost always use ``result`` not ``out`` but not a big deal. 

we NEVER use an ``assert_numpy_array_equal`` unles you are explicity comparing numpy arrays. Since you are comparing frames use ``assert_frame_equal``